Add WP Remote User Sync plugin and options

stronenv opened the pull request:
Added the [WP Remote User Sync](https://wordpress.org/plugins/wp-remote-users-sync/) to deny list and the plugin options to scrub list.

Hey @NickGreen! It looks like the plugin stores all options in a `wprus` array, is there a way to only scrub the encryption keys?

stronenv committed:
Add WP Remote User Sync plugin and options

Added the WP Remote User Sync to deny list and the plugin options to scrublist.

NickGreen commented:
> Added the [WP Remote User Sync](https://wordpress.org/plugins/wp-remote-users-sync/) to deny list

Could you please add it in alphabetical order? :) 

> It looks like the plugin stores all options in a wprus array, is there a way to only scrub the encryption keys?

There is definitely a way, but we usually only do that if we need to. For example, if clearing out the whole option causes fatals, or like in the case of Stripe, we want to keep the test API keys in place. 

If there's a good reason to only clear the keys, I'm happy to help you add that ([an example of how we do that for another plugin is here](https://github.com/a8cteam51/safety-net/blob/7d995225bee323dc69f1fd070ad9b5a6219fb59d/includes/deactivate-plugins.php#L88)), just let me know.

stronenv committed:
Adding plugin and option in alphabetical order

stronenv committed:
Clear only encryption keys for WP Remote Users Sync

Clear some WP Remote Users Sync options to disable only keys needed for remote connections and keep the remaining settings intact.

stronenv commented:
Thanks for your feedback, @NickGreen! 

I've added two commits. Let me know what you think.

NickGreen commented:
@stronenv This looks good and clean. Did you test it? 

Disregard! I see in Slack you said:

> I’ve tested on local

So, I think we're good to go. I'll merge it.

NickGreen committed:
PHPCS fixes

NickGreen committed:
Bumped version

NickGreen reviewed (approved):
LGTM! I ran it through PHPCS, and bumped the version. I'll merge it.
